apps/user-office-backend/src/datasources/stfc/StfcProposalDataSource.ts

```diff
@@ -57,7 +57,9 @@ export default class StfcProposalDataSource extends PostgresProposalDataSource {
         if (filter?.text) {
           query
             .where('title', 'ilike', `%${filter.text}%`)
-            .orWhere('abstract', 'ilike', `%${filter.text}%`);
+            .orWhere('proposal_id', 'ilike', `%${filter.text}%`)
+            .orWhere('proposal_status_name', 'ilike', `%${filter.text}%`)
+            .orWhere('instrument_name', 'ilike', `%${filter.text}%`);
```

Comment:
I just thought I would mention here, but it's up to you whether you want to do this in this PR or not - Emma said in the last sprint that it would be useful to additionally filter proposals by PI name and email address.

I've opened https://github.com/UserOfficeProject/user-office-project-issue-tracker/issues/913, feel free to complete it as part of this or no worries if you think it's out of scope and best left. 🙂

Reply:
Okay will include the changes.

Reply:
It may require a lot of changes will pick up https://github.com/UserOfficeProject/user-office-project-issue-tracker/issues/913 separately.

Reply:
Ok sure. Don't pick up the other one though as it's very very low priority :)